ci/docker_push.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -ev
+want_push='false'
+for branch in "master" "1.0.0" "1.1.0"
+do
+   if [ "$TRAVIS_BRANCH" == "$branch" ]
+   then
+       want_push='true'
+   fi
+done
+# if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$want_push" == "true" ] && [ "$TEST_TYPE" == "normal" ]
+if [ 0 == 0 ]
+then
+    # docker login -e $DOCKER_EMAIL -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+    export TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH; fi`
+    echo "TAG is $TAG"
```

Comment:
The 1.0.0 and 1.1.0 stuff was copied from somewhere else. We will delete that. I have no preference on whether we call master unstable or latest. Whatever folks want is fine.

I think for now, as long as we are building on every master commit, we will end up manually tagging each release, and can also point latest at that release. (Need docs on this, and eventually potentially a script for doing an official release).